.github/workflows/docs-site.yml

```diff
@@ -0,0 +1,94 @@
+# Generates and deploys static site to GitHub Pages
```

Comment:
🤔 Thinking if I am forgetting to update any existing code/script that may be related to it.